prac_05/word_occurrences.py

```diff
@@ -0,0 +1,27 @@
+# Adrianto Hartono
+# 14347930
+
+"""
+Word Occurrences
+Estimate: 30 minutes
+Actual: 40 minutes
+"""
+
+user_input= str(input("Text: "))
```

Comment:
No need to put str before input because anything that is inputted by the user would automatically be recognized as string